tests/test_data/synthetic_windiris_generator.py

```diff
@@ -55,6 +55,7 @@ def main():
             dfd.index.map(lambda a: a.strftime('%Y-%m-%d %H:%M:%S.%f')))
 
         fname = os.path.join(root, date.strftime('WI_%Y-%m-%d.csv'))
+        os.makedirs(root, exist_ok=True)

```

Comment:
Maybe you should check if it is a file before creating. But it's not very important